src/reyna/Connection/ConnectionInfo.cs

```diff
@@ -62,6 +62,62 @@ public bool Mobile
             }
         }
 
+        public bool Wifi
+        {
+            get
+            {
+                try
+                {
+                    var interfaces = NetworkInterface.GetAllNetworkInterfaces();
+                    foreach (var ni in interfaces)
+                    {
+                        if (WifiNetwork(ni))
+                        {
+                            return true;
+                        }
+                    }
+                }
+                catch (Exception)
+                {
+                }
+
+                return false;
+            }
+        }
+
+        public bool Roaming
+        {
+            get
+            {
+                try
+                {
+                    var interfaces = NetworkInterface.GetAllNetworkInterfaces();
+                    foreach (var ni in interfaces)
```

Comment:
Same here, not really relevant if we are roaming, but at the same time also on a wifi network, so not actually using the roaming wwan network for data transfer. Should only matter if the active network is roaming.